kinto2xml/amo.py

```diff
@@ -16,6 +16,9 @@ def prepare_amo_records(records, fields):
         for field in list(record.keys()):
             if field not in fields:
                 del record[field]
```

Comment:
please add some comment: what are the fields you do not want to keep ? what is the spec ? is this list finite like kinto fields like `id` and `last_modified` ?